### ISSUE TITLE
Missing 7z executable crashes CitizenOS API on container download

### ISSUE THREAD

**Issue by moll** [citizenos/citizenos-api#66]
Seems that a missing `7z` executable crashes the app on container download:
```
Error: spawn 7z ENOENT
    at exports._errnoException (util.js:1020:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:197:32)
    at onErrorNT (internal/child_process.js:376:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickDomainCallback (internal/process/next_tick.js:128:9)
```

While there is a `7z` exe in the `bin` folder, using it requires remembering to set the `PATH` before starting the server. Crashing, however, is probably not a good outcome in any situation.

**Comment by anettlinno** [comment]
Triage 58. Low priority issue as we haven´t received any feedback from API users after Dec 2018. Closing the issue with no further action.
